### ISSUE TITLE
Add JSDocs or other type of documentation.

### ISSUE THREAD

**Issue by SwapnikKatkoori** [SwapnikKatkoori/Text-Summarizer#8]
The code could really use some documentation to be easier to maintain.

**Comment by caelinsutch** [comment]
Is this project still maintained?

**Comment by SwapnikKatkoori** [comment]
As of right now, not really :(

**Comment by caelinsutch** [comment]
I'm probably going to start working with this, as of now everything is working properly? Are there typescript definitions written as well? I'll probably write some JSdocs and typescript definitions if needed.

**Comment by SwapnikKatkoori** [comment]
Yes, as of right now I believe everything works except for a bug that a user found in wordpos(issue 12). I haven't really had a chance to look at this yet.
